### PR TITLE
Feature ReCaptcha2

### PR DESCRIPTION
**Pull request by savolkov**

### CODE REVIEW

File: addon/components/flexberry-recaptcha.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+      window.grecaptcha.reset(this.get('widgetId'));
+    }
+  },
+
+  /**
+    Allows to use onSuccess action.
+  */
+  successCallback(reCaptchaResponse) {
+    let action = this.get('onSuccess');
+    if (Ember.isPresent(action)) {
```
Comment (ehaberev) [inline addon/components/flexberry-recaptcha.js]: The type of `action` also should be checked.

File: addon/components/flexberry-recaptcha.js [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+      action(reCaptchaResponse);
+    }
+  },
+
+  /**
+    Allows to use onExpired action.
+  */
+  expiredCallback() {
+    let action = this.get('onExpired');
+    if (Ember.isPresent(action)) {
```
Comment (ehaberev) [inline addon/components/flexberry-recaptcha.js]: The type of `action` also should be checked.

File: blueprints/ember-flexberry-account/index.js [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/* globals module */
+module.exports = {
+  afterInstall: function() {
+    var _this = this;
+    return _this.insertIntoFile(
```
Comment (ehaberev) [inline blueprints/ember-flexberry-account/index.js]: It seems we shoud also include `<script src='https://www.google.com/recaptcha/api.js'></script>` on installing addon.

File: addon/controllers/login.js [hunk 4]
```diff
@@ -51,6 +51,9 @@ export default Ember.Controller.extend({
     },
     pwdReset: function() {
       this.transitionToRoute('pwd-reset');
+    },
+    Test(reCaptchaResponse) {
```
Comment (ehaberev) [inline addon/controllers/login.js]: We use lowerCamelCase for naming properties, methods and actions.

File: app/templates/login.hbs [hunk 5]
```diff
@@ -7,6 +7,9 @@
   {{t "forms.login.password-label"}}<br/>
   {{input type="password" value=password}}
 </div>
+<div class="captcha-div">
+  {{flexberry-recaptcha onSuccess=(action "Test")}}
```
Comment (ehaberev) [inline app/templates/login.hbs]: We use lowerCamelCase for naming properties, methods and actions.

File: blueprints/ember-flexberry-account/index.js (hunk 3 above)
Comment (ehaberev) [inline blueprints/ember-flexberry-account/index.js]: Also we shoud add `ENV.recaptcha.siteKey` option (with empty value) and `ENV.contentSecurityPolicy` options for recaptcha in `environment.js` on installing addon.

Comment (savolkov) [reply blueprints/ember-flexberry-account/index.js]: https://github.com/Flexberry/ember-flexberry-account/blob/be64939b1e9a33972fe7ab2155faf00b053df63f/tests/dummy/app/index.html#L14

**Pull request merged**